app/views/layouts/application.html.erb

```diff
@@ -14,19 +14,16 @@
       <% if session[:current_user_id] %>
         <%= render  partial: "/layouts/sidebar" %>
       <% end %>
-      <% if flash[:error] %>
-        <%= render partial: "/layouts/flash-message" %>
-      <% end %>
-      <% if flash[:notice] %>
-        <%= render partial: "/layouts/flash-message" %>
-      <% end %>
       <div class="flex flex-col w-0 flex-1 overflow-hidden">
         <main
           class="flex-1 relative z-0 overflow-y-auto focus:outline-none"
           tabindex="0"
         >
           <div class="py-6">
             <div class="max-w-7xl mx-auto px-4 sm:px-6 md:px-8 mt-10 md:mt-0">
+              <% if flash[:error] || flash[:notice] %>
+                <%= render partial: "/layouts/flash-message" %>
+              <% end %>
```

Comment:
I was thinking it would be best if it appear as notification bar. It is shifting the form and other content down. Also there should be cross icon for removing the notification

Reply:
Removed Unnecessary spaces 🤔 I hope it would look good. 📍

Reply:
I appreciate for dedication to your very good quality work. Good job. But unfortunately i am unable to see your PR branch. But i trust your words. Thanks

Reply:
🤩

No no, moved to new PR. This branch gets polluted after accident merge 😪